code/game/antagonist/outsider/ert.dm

```diff
@@ -29,6 +29,7 @@ var/datum/antagonist/ert/ert
 	player.current << "You should first gear up and discuss a plan with your team. More members may be joining, don't move out before you're ready."
 
 /datum/antagonist/ert/equip(var/mob/living/carbon/human/player)
+	var/mob/living/M = player
```

Comment:
It shouldn't be, not in this case.

var/player is already mob/living/carbon/human, which is a child of mob/living (var/M), as such, it already has all of the variables inherited from var/M. Was var/player defined as mob/living, and var/M defined as mob/living/carbon/human, then it would be necessary.